extensions/data-workspace/src/dataworkspace.d.ts

```diff
@@ -68,6 +68,11 @@ declare module 'dataworkspace' {
 		 * Gets the supported project types
 		 */
 		readonly supportedProjectTypes: IProjectType[];
+
+		/**
+		 * Gets the extension ID for the project provider
+		 */
+		readonly providerExtensionId: string;
```

Comment:
Is there a better way to get the full extension name (e.g. "Microsoft.sql-database-projects") without depending on the extension to implement this correctly?

Reply:
I'll confirm that this is the case via fiddler, but I think that would always include the `data-workspace` extension ID if it's coming from this extension.  What I'm meaning to capture with this is the set of project-contributing extensions that have registered with `data-workspace`.

Reply:
Confirmed: the extension ID that's automatically captured isn't the one I'm intending to here.